src/documentation/dsls/sql/shell.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: section
+title: "Beam SQL: Shell"
+section_menu: section-menu/sdks.html
+permalink: /documentation/dsls/sql/shell/
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Beam SQL Shell
+
+## Overview
+
+Starting with version 2.6.0, Beam SQL includes an interactive shell, called the Beam SQL shell. The shell allows you to write pipelines as SQL queries without needing the Java SDK. By default, Beam uses the `DirectRunner` to execute the queries as Beam pipelines.
```

Comment:
nit: Should we make this `2.7.0`? These instructions won't work in `2.6.0`.

Reply:
These instructions should work as of late June. 2.6.0 RC was only cut last week.